### PR TITLE
LPS-164999 display chart as table

### PR DESCRIPTION
**Pull request by gustavolimav**
https://issues.liferay.com/browse/LPS-166816
https://issues.liferay.com/browse/LPS-166817
https://issues.liferay.com/browse/LPS-166818

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java [hunk 1]
```diff
@@ -1280,6 +1280,19 @@ public boolean isDisabledManagementBar() {
...
 	}
 
+	public boolean isDisplayChartAsTable() throws PortalException {
+		DDMFormInstance ddmFormInstance = getDDMFormInstance();
+
+		if (ddmFormInstance == null) {
+			return false;
+		}
+
+		DDMFormInstanceSettings settingsModel =
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java:1290]: Please, rename to `ddmFormInstanceSettings`

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
@gustavolimav he is going to fix 1 test but the code is okay :+1:

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
@marcelabc Could I forward this directly to bchan?

**Comment by gustavolimav** [comment]
Forward manually: https://github.com/brianchandotcom/liferay-portal/pull/125624

**Comment by carolmariaabb** [comment]
ci:close

**Pull request closed without merging**